### PR TITLE
Desktop

### PR DESCRIPTION
**Pull request by jburel**
This PR allows to run neo4j community in the browser.
The database is populated during the initialisation procedure using the CSV files profided.
To check:
* start either locally or on mybinder.org
* when ready click on ``new > desktop``
Screenshot 2021-05-12 at 13 26 56
* When the desktop is up, click on the browser icon
* in firefox tab enter ``localhost:7474``
* Enter user name and password 
* Explore the data

Screenshot 2021-05-12 at 13 30 08

cc @YojanaGadiya @agiani99

### CODE REVIEW

**Comment by YojanaGadiya** [comment]
Hey, I was trying to launch the desktop but I do not find the desktop option.. 
image

**Comment by jburel** [comment]
This means that the noVnc did not get installed for some reasons.
I had to change url recently since the version on bintray has been removed.
did you build locally? if so you should have a warning in the console

**Comment by jburel** [comment]
Did you build from the ``desktop`` branch? I cannot reproduce the issue

**Comment by jburel** [comment]
Screenshot 2021-05-12 at 18 02 58
This the build on mybinder.org

**Comment by YojanaGadiya** [comment]
Yes, I did do it from the desktop branch. Let me try again..

**Comment by YojanaGadiya** [comment]
The issue remains. I use the my_binder link in the README file in your forked repository; the desktop branch. It loads the notebook and but no desktop option available. No, I have not run it locally, but I can give it a try tomorrow and see what issues come up.

**Comment by jburel** [comment]
I see, the link will be working when this PR is merged.
For now, use 
https://mybinder.org/v2/gh/jburel/Neo4J-graph/desktop?filepath=notebooks

File: binder/postBuild [hunk 1]
```diff
@@ -10,13 +53,29 @@ mv neo4j-community-${VERSION} neo4j-community
...
-echo "dbms.connectors.default_listen_address=0.0.0.0" >> ${CONDA_DIR}/neo4j-community/conf/neo4j.conf
 
 # neo4j has a default password: neo4j
 # However, if you log in with that, the session will soon time out
 # and you will be required to update the original default password 
 ${CONDA_DIR}/neo4j-community/bin/neo4j-admin set-initial-password neo4jbinder
+
 cp -R data ${CONDA_DIR}/neo4j-community/import
+
+# install bloom
```
Comment (YojanaGadiya) [inline binder/postBuild:64]: Would we still need the bloom installations?

**Comment by YojanaGadiya** [comment]
Also, can I merge this branch into master? I will change the scripts which need to run for that purpose after the merge so that the docker does not break.

File: binder/postBuild (hunk 1 above)
Comment (jburel) [reply binder/postBuild:64]: Let me push one commit to remove that.
then we can merge

Comment (jburel) [reply binder/postBuild]: removed

**Comment by jburel** [comment]
@YojanaGadiya I have removed bloom since we no longer need it. Ready to merge when GitHub Action is green
